projects/plugins/jetpack/extensions/blocks/recurring-payments/test/fixtures/jetpack__recurring-payments--with-color-attrs__deprecated-1.serialized.html

```diff
@@ -1,3 +1,5 @@
-<!-- wp:jetpack/recurring-payments {"planId":228,"align":"center"} -->
-<div class="wp-block-jetpack-recurring-payments"><!-- wp:jetpack/button {"element":"a","uniqueId":"recurring-payments-id","text":"$0.50 Contribution","textColor":"green","backgroundColor":"purple"} /--></div>
+<!-- wp:jetpack/recurring-payments -->
+<div class="wp-block-jetpack-recurring-payments"><!-- wp:jetpack/recurring-payments-button {"planId":228,"align":"center"} -->
```

Comment:
Shouldn't the parsed html not contain these attributes? The block's index.js now doesn't expose any attributes.

Reply:
wp:jetpack/recurring-payments doesn't have attributes. wp:jetpack/recurring-payments-button does have attributes